Moq.AutoMock/AutoMocker.cs

```diff
@@ -297,6 +294,21 @@ public void Use<TService>(Expression<Func<TService, bool>> setup)
             return null!;
         }
 
+        /// <summary>
+        /// Searches and retrieves an object from the container that matches TService. This can be
+        /// a service setup explicitly via `.Use()` or implicitly with `.CreateInstance()`.
+        /// </summary>
+        /// <typeparam name="TService">The class or interface to search on</typeparam>
+        /// <param name="enablePrivate">When true, private constructors will also be used to create mocks.</param>

```

Comment:
nit: I realize we went with the `enablePrivate` because that matches previous APIs. However in the xml docs it would be nice to correct this to be a bit more accurate. 
```suggestion
        /// <param name="enablePrivate">When true, non-public constructors will also be used to create mocks.</param>

```